dlpy/speech_utils.py

```diff
@@ -399,3 +401,407 @@ def clean_audio(listing_path_local, segment_path_local_list):
             is_removed = True
     if is_removed:
         print("Note: all temporary files are removed.")
+
+
+def random_file_from_dir(local_audio_file):
+    n=0
+    random.seed()
+    for r, d, f in os.walk(local_audio_file):
+      for name in f:
+        n=n+1
+        if random.uniform(0, n) < 1:
+            random_file=os.path.join(r, name)
+    return random_file
+
+
+def play_one_audio_file(local_audio_file):
+    '''
+    Play a local audio file using soundfile and sounddevice.
+
+    Parameters
+    ----------
+    local_audio_file : string
+        Local location to the audio file to be played. When it is a directory,
+        a file will be randomly chosen.
+
+    Returns
+    -------
+    None
+
+    Raises
+    ------
+    DLPyError
+        If anything goes wrong, it complains and prints the appropriate message.
+
+    '''
+
+    try:
+        import soundfile as sf
+        import sounddevice as sd
+    except (ModuleNotFoundError, ImportError):
+        raise DLPyError('cannot import soundfile or sounddevice')
```

Comment:
can we please add one more note indicating that they might need to add those packages